lib/stores/MemcachedCacheStore.js

```diff
@@ -8,13 +8,22 @@ module.exports = class MemcachedCacheStore extends CacheStore {
     const location = opts.location || 'localhost:11211';
     this.memcached = new Memcached(location);
 
+    this.cacheKeyPrefix = opts.cacheKeyPrefix;
+    this.defaultTtlInSeconds = opts.defaultTtlInSeconds || 10 * 60;
+
     this._queuedKeys = [];
     this._scheduledMultiGet;
   }
 
+  _toCacheKey(key) {
+    return this.cacheKeyPrefix ?
+      [this.cacheKeyPrefix, key].join(':') :
```

Comment:
you could use templates here still. just not on client side code. this is fine too